.appveyor.yml

```diff
@@ -0,0 +1,36 @@
+skip_branch_with_pr: true
+
+clone_folder: c:\projects\qpentago
+
+image: Visual Studio 2017
+
+init:
+    - git config --global core.autocrlf input
+
+matrix:
+    fast_finish: false
+
+platform:
+    - x64
+
+configuration:
+    - RelWithDebInfo
+
+environment:
+    matrix:
+        - TOOLCHAIN: msvc15
+
+install:
+    - cd %APPVEYOR_BUILD_FOLDER%
```

Comment:
according to [build-pipeline](https://www.appveyor.com/docs/build-configuration/#build-pipeline), this is not required